js/dactylo.js

```diff
@@ -1,4 +1,6 @@
-const g30Keys = [
+const STARTING_LEVEL = 4;
+const MIN_WORD_WOUNT = 42;
```

Comment:
The fuck is a "wount" ?

Reply:
Totally your fault. It should be `COUNT` but I’m getting confused between Lafayette and Ergo‑L.